I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 69, in script call
    call consentscene
  File "game/consentscene.rpy", line 30, in script call
    call endingresult
  File "game/endingresult.rpy", line 67, in script
    jump credits
ScriptError: could not find label 'credits'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 69, in script call
    call consentscene
  File "game/consentscene.rpy", line 30, in script call
    call endingresult
  File "game/endingresult.rpy", line 67, in script
    jump credits
  File "renpy/ast.py", line 1752, in execute
    rv = renpy.game.script.lookup(target)
  File "renpy/script.py", line 918, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'credits'.

Windows-7-6.1.7601-SP1
Ren'Py 7.4.8.1895
Tri City Monsters 1.0
Fri Oct  1 01:14:34 2021
